tests/test_data/test_data_for_D0.json

```diff
@@ -9,12 +9,12 @@
             "value": 0.07264891149004721
         },
         "currency": "NOK",
-        "discount_factor": {
+        DISCOUNTFACTOR: {
```

Comment:
`"discount_factor"` should be replaced by `DISCOUNTFACTOR` only in `.py` files, not in `.json` files